projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Quotes.php

```diff
@@ -450,7 +450,7 @@ public function save_data( $quoteID, $quote ) {
 
                         #} Save content
                         //$data=htmlspecialchars($_POST['zbs_quote_content'], ENT_COMPAT);
-                        $quote['content'] = zeroBSCRM_io_WPEditor_WPEditorToDB($_POST['zbs_quote_content']);                        
+						$quote['content'] = wp_kses( wp_unslash( $_POST['zbs_quote_content'] ), $zbs->acceptable_html ); // phpcs:ignore WordPress.Security.NonceVerification.Missing -- to follow up with.
```

Comment:
I don't think we should use `wp_unslash()` here either.

Reply:
Ah yes, same issue.